src/dom.js

```diff
@@ -16,6 +16,35 @@
 
 
 /**
+ * Whether the element is currently contained in the DOM. Polyfills
+ * `document.contains()` method when necessary.
```

Comment:
Added "inclusionary" in docs and also added tests to ensure this on polyfill. But in the end I still had to go with Malte's recommendation, because we still need to be able to do `document.contains(document)` :)